cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/BuildCommand.java

```diff
@@ -198,6 +198,13 @@ public BuildCommand() {
             "generation")
     private String graalVMBuildOptions;
 
+    @CommandLine.Option(names = "--optimize", description = "generate optimized executable jar", defaultValue = "false")
+    private Boolean optimizeCodegen;
+
+    @CommandLine.Option(names = "--optimize-report", description = "generate codegen optimization reports",
```

Comment:
```suggestion
    @CommandLine.Option(names = "--optimize-report", description = "generate code generation optimization reports",
```

Reply:
Fixed in 7a9bcba1a96620fe099b7fe54a4fe8dc18ffb1ac.